source/tutorial/routes-and-templates.md

```diff
@@ -323,30 +329,34 @@ such as visiting routes, filling in fields, clicking on links/buttons, and waiti
 
 Some of the helpers we'll use commonly are:
 
-* [`visit`](http://emberjs.com/api/classes/Ember.Test.html#method_visit) - loads a given URL
-* [`click`](http://emberjs.com/api/classes/Ember.Test.html#method_click) - pretends to be a user clicking on a specific part of the screen
-* `andThen` - waits for our previous commands to run before executing our function.
-  In our test below, we want to wait for our page to load after `click` is called so that we can double-check that the new page has loaded
-* [`currentURL`](http://emberjs.com/api/classes/Ember.Test.html#method_currentURL) - returns the URL of the page we're currently on
+* [`visit`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#visit) - loads a given URL
+* [`click`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#click) - pretends to be a user clicking on a specific part of the screen
+* [`currentURL`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#currenturl) - returns the URL of the page we're currently on
+
+Let's import these helpers into our acceptance test:
+
+```/tests/acceptance/list-rentals-test.js
+import {
+  click,
+  currentURL,
+  visit
+} from '@ember/test-helpers'
+```
 
 ### Test visiting our About and Contact pages
 Now let's add code that simulates a visitor arriving on our homepage, clicking one of our links and then visiting a new page.
 
-```/tests/acceptance/list-rentals-test.js{+2,+3,+4,+5,+6,+10,+11,+12,+13,+14}
-test('should link to information about the company.', function (assert) {
-  visit('/');
-  click('a:contains("About")');
-  andThen(function() {
-    assert.equal(currentURL(), '/about', 'should navigate to about');
-  });
+```/tests/acceptance/list-rentals-test.js{+2,+3,+4,+8,+9,+10}
+test('should link to about page', async function(assert) {
+  await visit('/');
+  await click("a[href='/about']");
```

Comment:
I guess you are aware that this is not exactly best practice :wink:

Reply:
struggling with the best practice here... I used to do `click('a:contains("About")'` which no longer works.   Anyways open to better practices.

Reply:
best practice is using something like https://github.com/simplabs/ember-test-selectors 😉

Reply:
yeah, agree.  Just hesitating on bringing another addon into the tutorial....

Reply:
@toddjordan may be add a note at the end of this page of the guides mentioning the [testing section](https://emberobserver.com/categories/testing) of ember observer as a way to further improve the testing experience?

Reply:
> Just hesitating on bringing another addon into the tutorial....

I do agree on that.

We could still use the `data-test-foobar` attributes though since they do work without the addon. The main thing that the addon does is stripping those attributes out from the production build to save bytes.

But in the end it would be one additional thing to teach, so maybe not ideal in this context.